bundles/org.openhab.ui.homebuilder/web/src/definitions.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * i18n definitions
+ */
+export var languages = [
+  { name: 'English', id: 'en-UK' },
+  { name: 'French', id: 'fr-FR' },
+  { name: 'German', id: 'de-DE' },
+  { name: 'Italian', id: 'it-IT' },
+  { name: 'Polish', id: 'pl-PL' },
+  { name: 'Russian', id: 'ru-RU' },
+  { name: 'Spanish', id: 'es-ES' }
+];
+
+/**
+ * Structure definitions
+ */
+export var floors = [
+  { abbr: 'GF', value: 'GroundFloor', icon: 'groundfloor' },
+  { abbr: 'FF', value: 'FirstFloor', icon: 'firstfloor' },
+  { abbr: 'F2', value: 'SecondFloor', icon: 'attic' },
+  { abbr: 'F3', value: 'ThirdFloor', icon: 'attic' },
+  { abbr: 'F4', value: 'FourthFloor', icon: 'attic' }
+];
+
+export var rooms = [
+  { value: 'Attic', icon: 'attic' },
+  { value: 'Balcony', icon: '' },
+  { value: 'Backyard', icon: 'grass' },
+  { value: 'Basement', icon: 'cellar' },
+  { value: 'Bathroom', icon: 'bath' },
+  { value: 'Bedroom', icon: 'bedroom' },
+  { value: 'Boiler', icon: 'boiler_viessmann' },
+  { value: 'Wardrobe', icon: 'wardrobe' },
+  { value: 'Cellar', icon: 'cellar' },
+  { value: 'Corridor', icon: 'corridor' },
+  { value: 'Deck', icon: '' },
+  { value: 'Dining', icon: '' },
+  { value: 'Downstairs', icon: 'cellar' },
+  { value: 'Driveway', icon: 'garage_detached' },
+  { value: 'Entryway', icon: 'frontdoor' },
+  { value: 'FamilyRoom', icon: 'parents_2_4' },
+  { value: 'FrontYard', icon: 'grass' },
+  { value: 'Garage', icon: 'garage' },
+  { value: 'GuestHouse', icon: 'house' },
+  { value: 'GuestRoom', icon: 'parents_4_3' },
+  { value: 'Hallway', icon: 'corridor' },
+  { value: 'HomeCinema', icon: 'television' },
+  { value: 'KidsRoom', icon: 'girl_3' },
+  { value: 'Kitchen', icon: 'kitchen' },
+  { value: 'LaundryRoom', icon: 'washingmachine' },
+  { value: 'Library', icon: 'office' },
+  { value: 'LivingRoom', icon: 'sofa' },
+  { value: 'LivingDining', icon: 'sofa' },
+  { value: 'Loft', icon: 'attic' },
+  { value: 'Lounge', icon: 'sofa' },
+  { value: 'MasterBedroom', icon: 'bedroom_red' },
+  { value: 'NannyRoom', icon: 'woman_1' },
+  { value: 'Office', icon: 'office' },
+  { value: 'Outside', icon: 'garden' },
+  { value: 'Patio', icon: 'terrace' },
+  { value: 'Porch', icon: 'group' },
+  { value: 'Stairwell', icon: 'signal' },
+  { value: 'StorageRoom', icon: 'suitcase' },
+  { value: 'Studio', icon: 'pantry' },
+  { value: 'Shed', icon: 'greenhouse' },
+  { value: 'Toilet', icon: 'toilet' },
+  { value: 'Terrace', icon: 'terrace' },
+  { value: 'Upstairs', icon: 'firstfloor' }
+];
+
+export var devices = [
```

Comment:
"objects" seem to be too vague, especially in the JavaScript context. I could go with "smartObjects" or "smartThings" or maybe just "things"? Would like to avoid confusion too.

Reply:
"Things" is already the word for the physical side of openHAB, definitely not the right term for items.
"smart" is also not correct as your windows is not smart.
Check out https://youtu.be/R1HVYWP51E0. I tentively used "BuildingObjects" there, but that is only valid for windows & doors and alike. But if you want to cover also furniture, lights, tvs, etc. "Object" is the only reasonable term, really.